signalfx/client.py

```diff
@@ -49,6 +56,25 @@ def send_event(self, event_type, dimensions=None, properties=None):
         logging.debug('Sending event to SignalFx: %s', data)
         return data
 
+    @staticmethod
+    def get_aws_unique_id(timeout=DEFAULT_TIMEOUT):
```

Comment:
I don't think its a good idea to use the default timeout. 

So the default timeout is made for the post call to signalfx, while this AWS call is an internal call. Using same timeout for both might be inconvenient when someone wants to break it up.

Given the above, you should probably change the DEFAULT_TIMEOUT to something like HTTP_REQUEST_TIMEOUT.